### PR TITLE
Barulin lab 1

### PR DESCRIPTION
**Pull request by alexandrbarulin**
Added 7-8 lab

### CODE REVIEW

**Comment by DmytroOlegovychDev** [comment]
Like

**Review by DmytroOlegovychDev — APPROVED** [review]
Like

File: labs/alexandr_barulin/lab_8/task_3.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""
+З множини чисел [1..n] виділити підмножину досконалих чисел (число
+називається досконалим якщо воно дорівнює сумі своїх власних дільників,
+наприклад 6=3+2+1)
+"""
+
+def FindTrueOrFalse(N):
```
Comment (sen-den) [inline labs/alexandr_barulin/lab_8/task_3.py:7]: Not the best name for this method :)

File: labs/alexandr_barulin/lab_8/task_1.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"""
+З множини чисел [1..n] виділити перетин підмножин - чисел виду 4k+1 і
+5k+4.
+"""
+
+C = int(input("Введите C: "))
+N = {i for i in range(C)}
+
+list1 = [4 * i + 1 for i in range(C)]
```
Comment (sen-den) [inline labs/alexandr_barulin/lab_8/task_1.py:9]: For N=10 it have to produce [1; 5; 9] -- not larger than 10

**Review by sen-den — APPROVED** [review]

**Pull request merged**